neutone_sdk/benchmark.py

```diff
@@ -0,0 +1,126 @@
+import timeit
+import itertools
+import click
+import torch
+from neutone_sdk.utils import load_neutone_model
+import numpy as np
+
+
+@click.group()
+def cli():
```

Comment:
A quick comment here why this method is required could be helpful since it seems useless at first glance.